vertical-pod-autoscaler/enhancements/4016-in-place-updates-support/README.md

```diff
@@ -254,6 +257,48 @@ tested in the following scenarios:
 * In-place update will fail but `CanEvict` is false, pod should not be evicted.
 * In-place update will fail but `EvictionRequirements` are false, pod should not be evicted.
 
+### Upgrade / Downgrade Strategy
+
+#### Upgrade
+
+On upgrade of the VPA to 1.4.0 (tentative release version), nothing will change,
+VPAs will continue to work as before.
+
+Users can use the new `InPlaceOrRecreate` by enabling the alpha Feature Gate (which defaults to disabled)
+by passing `--feature-gates=InPlaceVerticalScaling=true` to the recommender and admission-controller components and setting
```

Comment:
should be `updater` and `admission-controller`

Reply:
Whoops! Fixed.